### ISSUE TITLE
Release 1.6.5 is missing Module Folder

### ISSUE THREAD

**Issue by willemfrishert** [datatheorem/TrustKit#222]
I'm using Carthage to pull down TrustKit and noticed that in the 1.6.5 release has a different layout of the .framework folder than the 1.6.4 release before it.

I've created a test with 1.6.4 to switch to 1.6.5
1. Create a new xcode project
2. Create a Cartfile
3. Add `github "datatheorem/TrustKit" == 1.6.4` to the Cartfile
4. Add `$(PROJECT_DIR)/Carthage/Build/iOS` to the `FRAMEWORK_SEARCH_PATHS`
5. Run `carthage update --platform iOS --new-resolver`
6. Go back to xcode, open a swift file and add `import TrustKit`
and perform clean build of the project
7. Open up the Cartfile and change it `github "datatheorem/TrustKit" == 1.6.5`
8. remove the Carthage folder
9. Run `carthage update --platform iOS --new-resolver`
10. Go back to xcode, and perform clean build of the project

What happens:
- After step 5 the contents of `Carthage/Build/iOS/TrustKit.framework` appears as:
```
$ ls -al Carthage/Build/iOS/TrustKit.framework
total 4704
drwxr-xr-x  7 developer  staff      224 Mar 19 20:01 .
drwxr-xr-x  6 developer  wheel      192 Mar 19 20:01 ..
drwxr-xr-x  8 developer  staff      256 Mar 19 20:01 Headers
-rw-r--r--  1 developer  staff      740 Mar 19 20:01 Info.plist
drwxr-xr-x  3 developer  staff       96 Mar 19 20:01 Modules
-rw-r--r--  1 developer  staff        8 Mar 19 20:01 PkgInfo
-rwxr-xr-x  1 developer  staff  2398200 Mar 19 20:01 TrustKit
```
- At step 6, the TrustKit.framework is found via the `FRAMEWORK_SEARCH_PATHS` and `import TrustKit` works

After step 9:
```
$ ls -al Carthage/Build/iOS/TrustKit.framework
total 4704
drwxr-xr-x  6 developer  staff      192 Mar 19 20:10 .
drwxr-xr-x  6 developer  wheel      192 Mar 19 20:10 ..
-rw-r--r--  1 developer  staff      740 Mar 19 20:10 Info.plist
-rw-r--r--  1 developer  staff        8 Mar 19 20:10 PkgInfo
-rwxr-xr-x  1 developer  staff  2398188 Mar 19 20:10 TrustKit
drwxr-xr-x  8 developer  staff      256 Mar 19 20:10 public
```
After step 10:
- Xcode now reports `No such module 'TrustKit'` as the Modules/module.modulemap is missing

Details:
- carthage 0.34.0
- Xcode 11.1 (11A1027)
- Swift 5.1

Attachment:
I'm attaching my demo code where I got to step 9 in the reproduction steps: [TrustKit_Test_1.6.5.zip](https://github.com/datatheorem/TrustKit/files/4357821/TrustKit_Test_1.6.5.zip)

Steps to test
1. Download the binary;
2. Run `carthage update --platform iOS --new-resolver` (this will download 1.6.5);
3. Open the A.xcworkspace and perform clean build of the project.

Question:
Could you please advice how to import TrustKit 1.6.5 without module.modulemap?

I'm happy to provide additional feedback.

Cheers,
Willem

**Comment by finetuning-mg** [comment]
Same problem here: version 1.6.5 does not work with carthage and causes a "No such module 'TrustKit'" error in Xcode.
I am running carthage 0.34.0 and Xcode 11.3.1 (11C504) on macOS Mojave 10.14.6.
Version 1.6.4 works just fine.

**Comment by edwardbeecroft** [comment]
+1

**Comment by jsaini-hw** [comment]
+1

**Comment by kokluch** [comment]
+1

**Comment by nabla-c0d3** [comment]
I tried setting up Carthage after merging #230 (commit ec7cc7599559517d479265c9bf223e477ae5f88f) and it seemed to work.

**Comment by nabla-c0d3** [comment]
Released as v1.7.0.
